# HG changeset patch
# User Rik <rik@octave.org>
# Date 1350659499 25200
#      Fri Oct 19 08:11:39 2012 -0700
# Branch stable
# Node ID 69f45f3ab96ac14e1f530868f824e7d99d3a28c3
# Parent  d23eed1c5d5d03ed54bb2bf8bc2695c7e6632296
Correct big-endian/little-endian reversal in typecast documentation.

* typecast.cc(Ftypecast): Correct big-endian/little-endian reversal in typecast
documentation.

diff --git a/src/DLD-FUNCTIONS/typecast.cc b/src/DLD-FUNCTIONS/typecast.cc
--- a/src/DLD-FUNCTIONS/typecast.cc
+++ b/src/DLD-FUNCTIONS/typecast.cc
@@ -125,17 +125,17 @@ bit length of @var{x} is not divisible b
 occurs.\n\
 \n\
 An example of the use of typecast on a little-endian machine is\n\
 \n\
 @example\n\
 @group\n\
 @var{x} = uint16 ([1, 65535]);\n\
 typecast (@var{x}, 'uint8')\n\
-@result{} [   0,   1, 255, 255]\n\
+@result{} [   1,   0, 255, 255]\n\
 @end group\n\
 @end example\n\
 @seealso{cast, bitunpack, bitpack, swapbytes}\n\
 @end deftypefn")
 {
   octave_value retval;
 
   if (args.length () == 2)
